manage

```diff
@@ -417,6 +417,9 @@ ${DIRECTORY}/icons/none-1.png
   done
   #all actions have been completed. Daemon has effectively stopped listening, so remove its pid file
   rm -f "${DIRECTORY}/data/manage-daemon/pid"
+
+  #updates could have been run as part of the manage-daemon, so update the updatable-files and updatable-apps status files
+  "${DIRECTORY}/updater" set-status
```

Comment:
Is there a specific reason you have it check for updates between the current updates finishing and the error-explanation window opening? Could it be moved elsewhere or run as a background process?

Reply:
what error explaination window? this location happens after all install/uninstall/updates have fully completed. the error explanation window happens after each individual install/uninstall/update. that happens well before this gets run

Reply:
> what error explaination window? this location happens after all install/uninstall/updates have fully completed. the error explanation window happens after each individual install/uninstall/update. that happens well before this gets run

See the `diagnose_apps` line a few lines beneath this line. Current behavior is to accumulate errors and allow the user to click through all of them at the end of updates.

Reply:
ah my bad I thought from memory that diagnose_apps (the GUI) appeared immediatly after an app failure

I can move `set-status` after diagnose_apps. basically I want it to run after app updates are all complete otherwise the updatable_apps and updatable_files files do not get updated, meaning the "Update" row still appears in the GUI and the update GUI still shows these apps/files as updatable.

Reply:
resolved with https://github.com/Botspot/pi-apps/pull/2340/commits/80f7e4cc741792e9cf9460ccabe6f6f50c103837